tests/operator_test.go

```diff
@@ -1172,13 +1167,19 @@ var _ = Describe("ALL Operator tests", func() {
 					_, err := utils.FindPodByPrefix(f.K8sClient, f.CdiInstallNs, cdiDeploymentPodPrefix, common.CDILabelSelector)
 					return err
 				}, 2*time.Minute, 1*time.Second).Should(BeNil())
+
+				pod, err := utils.FindPodByPrefix(f.K8sClient, f.CdiInstallNs, cdiDeploymentPodPrefix, common.CDILabelSelector)
+				Expect(err).ToNot(HaveOccurred())
+
 				By("Ensuring this pod is the leader")
-				Eventually(func() bool {
-					controllerPod, err := utils.FindPodByPrefix(f.K8sClient, f.CdiInstallNs, cdiDeploymentPodPrefix, common.CDILabelSelector)
-					Expect(err).ToNot(HaveOccurred())
-					log := getLog(f, controllerPod.Name)
-					return checkLogForRegEx(logIsLeaderRegex, log)
-				}, 2*time.Minute, 1*time.Second).Should(BeTrue())
+				matchString := fmt.Sprintf("Successfully completed fsync(%s) syscall", common.WriteBlockPath)
```

Comment:
copy paste err? the fsync test is definitely not in operator_test.go

Reply:
That's right :sweat_smile: